### ISSUE TITLE
Training saturation issues

### ISSUE THREAD

**Issue by sir-avinash** [avisingh599/imitation-dagger#1]
Hello Avi,

I'm trying to implement your dagger.py code as is. When I run the code, I noticed that the loss saturates to 0.9453 in the first run itself (within the first few epochs), even before dagger iters start (in iter1 of dagger it satures to 1.0715). Can you mention what kind of losses you get at each run of your network? This would help in debugging. 

It is kind of frustrating that I'm unable to replicate your performance. I'm new to tensorflow-keras-python environments, but I'm very interested and your repo seemed like a good place to start tinkering. Do you suspect some network and torcs setup issues? Can you mention what versions of keras, torcs, etc you have used and on what hardware? I'd be grateful for any help in getting me out of this local minima :/

Quick Update: I changed the output activation to 'linear' and Adam's lr to 1e-3. This worked like a charm. It crashes at 134th step in iter1, 539th step in iter2, and the goes the distance. This was relieving :D However, it's still weird that I can't get the 'tanh' activation to work.
